src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -848,4 +848,31 @@
           }
         }
       }
+    },
+    "/restart": {
+      "get": {
+        "summary": "Restart the in-memory schema store.",
+        "operationId": "get_restart",
+        "consumes": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
+        "parameters": [],
+        "produces": ["application/vnd.schemaregistry.v1+json"],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
```

Comment:
Discussion point:

This returns and empty schema but I'm thinking we should actually return something instead that indicates if the restart was successful? I'm thinking something more than just HTTP exit codes.

Reply:
I think if restart fails it would be pretty catastrophic and internal server error is about all we could return.